### PR TITLE
add helix-js and set url / token

### PR DESCRIPTION
**Pull request by AlinaGoaga**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,6 +18,8 @@
     "@types/node": "^12.0.0",
     "@types/react": "^16.9.0",
     "@types/react-dom": "^16.9.0",
+    "add": "^2.0.6",
```
Comment (WebMaestroFr) [inline package.json:21]: Hum... did someone `yarn add add` here or something ?

File: package.json [hunk 2]
```diff
@@ -31,7 +33,8 @@
     "tslint-config-prettier": "^1.18.0",
     "tslint-plugin-prettier": "^2.3.0",
     "tslint-react": "^4.2.0",
-    "typescript": "~3.7.2"
+    "typescript": "~3.7.2",
+    "yarn": "^1.22.4"
```
Comment (WebMaestroFr) [inline package.json:37]: :|
`yarn add yarn add` ?!

File: src/components/Namespaces/Table/index.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+const NamespaceTable: FC = () => {
+  const [namespaces, setNamespaces] = useState<V1Namespace[]>();
+
+  useEffect(() => {
+    helix.namespaces.list().then((data) => setNamespaces(data.items));
+  }, []);
+
+  return (
+    <div>
```
Comment (WebMaestroFr) [inline src/components/Namespaces/Table/index.tsx:15]: This `div` is probably not necessary.

File: package.json (hunk 2 above)
Comment (AlinaGoaga) [reply package.json]: Removed :)

File: src/components/Namespaces/Table/index.tsx (hunk 3 above)
Comment (AlinaGoaga) [reply src/components/Namespaces/Table/index.tsx:15]: Removed :)

**Pull request merged**